CommitChange/houdini
# Allows nonprofits to have export formats for their payments export

Closes [#3349](https://github.com/CommitChange/tix/issues/3349).

Depends on #129.

To test, create a export_format:

```ruby
nonprofit = Nonprofit.find(<nonprofit_id>)
nonprofit.export_formats.create(
  name: 'CiviCRM format',
  date_format: 'MM/DD/YYYY',
  show_currency: false,
  custom_columns_and_values: {
    'payments.kind' => {
      'custom_values' => {
        'RecurringDonation' => 'Recurring Donation'
      },
      'custom_name' => 'Kind of Payment'
    }
  }
)
```

Navigate to the payments list and choose an export to run:

![image](https://user-images.githubusercontent.com/15739610/138970240-ff913ac8-0953-4b4d-b307-080bc6e1bce6.png)

You should see on the logs the resulting query.
